user/squashfs-tools/template.py

```diff
@@ -0,0 +1,25 @@
+pkgname = "squashfs-tools"
```

Comment:
does incus not work with the slightly differently named squashfs-tools-ng?

Reply:
No, it used the binary unsquashfs which is apparently different than rdsquashfs provided by squashfs-tools-ng.

Reply:
ok, makes sense. at first i thought it conflicted but all the bin names are unique so it's ok :)